aptos-move/framework/aptos-framework/sources/coin.spec.move

```diff
@@ -1,6 +1,29 @@
 spec aptos_framework::coin {
     spec module {
         pragma verify = true;
+        global supply<CoinType>: num;
+        global aggregate_supply<CoinType>: num;
+        apply TotalSupplyEqSum<CoinType> to *<CoinType> except
+            initialize, initialize_internal, initialize_with_parallelizable_supply;
+        apply TotalSupplyNoChange<CoinType> to *<CoinType> except mint,
+            burn, burn_from, initialize, initialize_internal, initialize_with_parallelizable_supply;
+    }
+
+    spec schema TotalSupplyEqSum<CoinType> {
+        invariant
+            (option::spec_is_some(global<CoinInfo<CoinType>>(type_info::type_of<CoinType>().account_address).supply)) ==>
```

Comment:
Could you define a spec function to shorten this making it more readable? For example, `spec_is_supply_tracked<CoinType>()`